deploy/operator_group.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  name: service-telemetry-operator-group
+  name: service-telemetry-og
```

Comment:
There was a reason this needed to be changed, but I can't remember why. The main reason this file shows `service-telemetry-operator-group` is because that's the name created when installing the Operator from OLM. If something else is referencing this name maybe we should adjust everything to align to `service-telemetry-operator-group` instead.

Reply:
in this case, it looked like it was forgotten to update this location.

Reply:
My instinct has always been to keep this matching the OLM generated name, so `service-telemetry-operator-group`. 

I'd like to either understand why it's being changed, or leave it unchanged.

Reply:
This comes from https://github.com/infrawatch/service-telemetry-operator/blob/master/build/stf-run-ci/tasks/setup_base.yml#L23

Reply:
Our official docs suggest the full name, though (Noticing it's a different API version as well):

```
Enter the following command:

oc apply -f - <<EOF
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: service-telemetry-operator-group
  namespace: service-telemetry
spec:
  targetNamespaces:
  - service-telemetry
EOF
```
https://infrawatch.github.io/documentation/#creating-an-operatorgroup

It's my understanding that if you install from the GUI (instead of the CLI steps in our docs) the operator group will also be created with the full name (though I haven't checked this recently).

I don't think this is a major issue, but it seems we are not being consistent. If we're changing this to fix a problem, let's document what the problem is. If we're changing it to be consistent, that's also good, but in this case I think we're changing it in the wrong direction and setup_base.yml is the odd one out that should be fixed.